### ISSUE TITLE
Can't call method from package

### ISSUE THREAD

**Issue by redcurse** [skycoin/cx#28]
When trying to call a method associated to a structure in a different package I get an error.

```
package Package

type Structure struct {
    string str
}

func (structure Structure) function() {
    str.print(structure.string)
}

package main
import "Package"

func main () {
    var structure Package.Structure
    structure.string = "abc"
    str.print(structure.string)
    structure.function()
}
```
Output
```
panic:

goroutine 1 [running, locked to thread]:
github.com/skycoin/cx/cxgo/actions.ProcessMethodCall(0xc00034e460, 0xc00007f170, 0xc0003072c0, 0xc00007f0a8, 0xc00007f201)
        C:/Users/Jay/go/src/github.com/skycoin/cx/cxgo/actions/cxgo_compiled.go:1897 +0x26d
github.com/skycoin/cx/cxgo/actions.FunctionDeclaration(0xc000193340, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0xc000004040, 0x4, 0x4)
        C:/Users/Jay/go/src/github.com/skycoin/cx/cxgo/actions/cxgo_compiled.go:2155 +0xfd7
main.yyParse(0xa46100, 0xc000300000, 0xc000300000)
        C:/Users/Jay/go/src/github.com/skycoin/cx/cxgo/cxgo.go:1714 +0x1b2a
main.main()
        C:/Users/Jay/go/src/github.com/skycoin/cx/cxgo/cxgo.nn.go:8362 +0x6d5
```

**Comment by amherag** [comment]
Methods are currently in pre-Alpha stage. Actually, the last commit I did related to methods was to fix a conflict between field accessing and methods, and it wasn't targeted at methods :(

I will definitely fix this, but it will have a medium priority. Give me some weeks. In the meantime, please use functions.

**Comment by redcurse** [comment]
I just submitted another issue related to methods. I just saw this afterwards. I'll leave the other issue but refrain from attempting to use methods for now.

Keep up the good work!

**Comment by amherag** [comment]
I just did a bit of rework with the fix to #29 and this also got fixed.
